app/src/forms/common/middleware/auth.js

```diff
@@ -58,7 +56,7 @@ const hasRole = (roles) => {
   }
 
   const rolecheck = (token, request) => {
-    const result = roles.some(r => token.hasRole(`${RESOURCE_ACCESS}:${r}`));
+    let result = roles.some(r => token.hasRole(`${resourceAccess}:${r}`));
```

Comment:
Result should still be a const as there is no other assignments to it in its scope.